_ml-commons-plugin/algorithms.md

```diff
@@ -316,4 +400,218 @@ The API responds with the sum of the contribution and base values per aggregatio
 
 ### Limitations
 
-The Localization algorithm can only be executed directly. Therefore, it cannot be used with the ML Commons Train and Predict APIs.
\ No newline at end of file
+The Localization algorithm can only be executed directly. Therefore, it cannot be used with the ML Commons Train and Predict APIs.
+
+## Logistic regression
+
+A classification algorithm, logistic regression models the probability of a discrete outcome given an input variable. In ML commons, these classifications include both binary and multi-class. The most common is the binary classification, which takes two values such as "true/false" or "yes/no", and predicts the outcome based on the values specified. On the other hand, a multi-class output can categorize different inputs based on type. This makes logistic regression most useful for situations where you are trying to determine how your inputs fit best into a specified-category. 
+
+### Parameters
+
+| Parameter | Type | Description | Default Value |
+|---|---|---|---|
+| learningRate | Double | The rate of speed the gradient moves during gradient descent | 1 |
+| momentumFactor | Double | The idea that a regressor that had recently risen or fallen during convergence will continue that trend over the medium term | 0 |
+| epsilon | Double | The criteria that a linear model is identified | 0.1 |
+| beta1 | Double | The exponential decay rates for the moment estimates | 0.9 |
+| beta2 | Double | The exponential decay rates for the moment estimates | 0.99 |
+| decayRate | Double | The Root Mean Squared Propagation (RMSProp) | 0.9 |
+| momentumType | MomentumType | The momentum with SDG to help accelerate gradients vectors in the right directions, leading to faster convergence between vectors | STANDARD |
+| optimizerType | OptimizerType | The optimizer used in the model  | AdaGrad |
+| target | String | The target field | null |
+| objectiveType | ObjectiveType | The objective function type | LogMulticlass |
+| epochs | Integer | The number of iterations | 5 |
+| batchSize | Integer | The size of minbatches | 1 |
```

Comment:
Is this properly "minibatches"?

Reply:
Yes it is minimum batches. @Naarcha-AWS I have updated the parameters descriptions in the quip doc. Can you please check and update them?